build.sbt

```diff
@@ -555,3 +556,12 @@ lazy val prometheusBackend: Project = (project in file("metrics/prometheus-backe
     )
   )
   .dependsOn(coreJVM)
+
+//-- http4s
+lazy val java11backend: Project = (project in file("java11-backend"))
+  .settings(commonJvmSettings: _*)
+  .settings(
+    name := "java11-backend",
+    scalacOptions ++= Seq("-J--add-modules", "-Jjava.net.http")
+  )
+  .dependsOn(catsJVM, coreJVM % compileAndTest)
```

Comment:
this module needs to be enabled only if the host-java is >= 11. Otherwise we need to skip it. Also, is it available for all Scala versions?

Reply:
+ on travis we have java 8. Is it possible to run a single test with another java?